example/pubspec.yaml

```diff
@@ -3,7 +3,7 @@ description: Demonstrates how to use the amplitude_flutter plugin.
 publish_to: 'none'
 
 environment:
-  sdk: ">=2.12.0 <3.0.0"
+  sdk: ">=3.3.0 < 4.0.0"
```

Comment:
Why 3.3 and not 3.0?

Reply:
Looking at the changelog, I see `js_interop` as early as 3.1, but there some breaking changes between 3.1 and 3.3, the most important one being [extension types ](https://dart.dev/language/extension-types). [This blogpost](https://medium.com/dartlang/history-of-js-interop-in-dart-98b06991158f) from dartlang says extension types and js_interop were officially released with 3.3 and are what enable WASM with flutter (main driving factor for ditching `package:js/js.dart`).